rpc/rpc.go

```diff
@@ -124,9 +125,15 @@ type ErrorReporter interface {
 	ReportError(error)
 }
 
-// NewConn creates a new connection that communications on a given
-// transport.  Closing the connection will close the transport.
-// Passing nil for opts is the same as passing the zero value.
+// NewConn creates a new connection that communicates on a given transport.
+//
+// Closing the connection will close the transport and release the bootstrap
+// client provided in opts.
+//
+// Passing nil for opts is the same as passing the zero value. This can be useful
+// for obtaining a reference to the remote side's bootstrap. This in turn allows
+// this listening connection to be used as the RPC client and make calls to the remote
+// side, since capnp connections are symmetric.
```

Comment:
I think these last two sentences have too little context to be useful, and the potential to be confusing (notably, passing in a bootstrap interface of your own doesn't stop you from calling .Bootstrap() on the connection to get the other side's bootstrap -- notably, Sandstorm does both when launching a grain).

I think there's often a temptation to try to stretch this into explaining more than really fits in the reference docs for one function, but what's really needed is better overview/tutorial docs.

Reply:
Yeah I can see your point. I found this such a valuable bit of insight that I put it in there, but yeah it fits better in documentation.
I've removed it when changing the API to take bootstrapClient instead of Options.